### ISSUE TITLE
about the label 

### ISSUE THREAD

**Issue by xiaoxiaowannote** [shufanwu/colorNet-pytorch#8]
hello ,thanks for your codes, i have been reading your codes recently,i have some trouble about the label,  can you show me some contents about the train.txt , and the label should be the name of classses of things in the image ? thanks , i am waiting for your reply.

**Comment by YONGSONGWAN** [comment]
is about the environment in the image as the paper saying?

**Comment by xiaoxiaowannote** [comment]
is about the environment in the image as the paper says?
